sdk/monitor/monitor-opentelemetry/src/metrics/handler.ts

```diff
@@ -56,9 +56,27 @@ export class MetricHandler {
       this._views.push({ meterName: "@azure/opentelemetry-instrumentation-redis" });
     }
     this._azureExporter = new AzureMonitorMetricExporter(this._config.azureMonitorExporterOptions);
+
+    // Determine export interval: prioritize OTEL_METRIC_EXPORT_INTERVAL env var, then options, then default
+    let exportIntervalMillis = this._collectionInterval;
+    let useEnvVar = false;
+
+    if (process.env.OTEL_METRIC_EXPORT_INTERVAL) {
```

Comment:
This check should not be here, all configuration related stuff should be in a single place, like existing config classes